src/components/replica/index.tsx

```diff
@@ -22,10 +21,8 @@ interface ReplicaElements {
   state?: React.JSX.Element;
 }
 
-const ReplicaDuration: FunctionComponent<{ created: Date; ended?: Date }> = ({
-  created,
-  ended,
-}) => {
+type DurationProps = { started: Date; ended?: Date };
+const ReplicaDuration = ({ started, ended }: DurationProps) => {
```

Comment:
Replica started is another field `startTime`. So technically it is `created`
https://github.com/equinor/radix-web-console/blob/ef27613438051031dfc87b62f1764e8603fd0209/src/components/replica/index.tsx#L137